### ISSUE TITLE
Use a second source for icons for assets

### ISSUE THREAD

**Issue by yabirgb** [rotki/rotki#4979]
Currently we relay on coingecko to query icons and due to the more restrictive rate limits querying icons can contribute to slower loads. As an alternative we can use the set of icons in https://tokenicons.org/ that aims for an open source resource

**Comment by LefterisJP** [comment]
The problem with that link is that it's extremely limited and does not have icons for other chains.


What's more with any such approach if there is any difference in quality, size, format of the image file or even the art for tokens we will have incosistency between the icons of the assets all around the app.

**Comment by kelsos** [comment]
> Currently we relay on coingecko to query icons and due to the more restrictive rate limits querying icons can contribute to slower loads. As an alternative we can use the set of icons in https://tokenicons.org/ that aims for an open source resource

We used to use [cryptocurrency-icons](https://www.npmjs.com/package/cryptocurrency-icons) in the past @yabirgb and it did work well for us.

As @LefterisJP mentioned inconsistency is one problem, the second is that there is no guarantee that the icon set will be properly maintained add updated in the future (this happened to us with the previous library at some point and we had to search for token icons to ship with the app every time we added a new token).

The reason we went with the current approach was because of the problems we had with this kind of icon sets.
